src/prefect/client/schemas/objects.py

```diff
@@ -90,6 +90,22 @@ class StateType(AutoEnum):
     CANCELLING = AutoEnum.auto()
 
 
+class StateName(AutoEnum):
+    SCHEDULED = "Scheduled"
```

Comment:
Same comment here about user-produced state names, I think this enum conflicts with that idea

Reply:
Makes sense! I've removed this in [ec46ace](https://github.com/PrefectHQ/prefect/pull/13152/commits/ec46ace810e1ce1357b5ff05d5b4d94f19f39e87)